pipenv/patched/piptools/resolver.py

```diff
@@ -120,7 +120,7 @@ def resolve(self, max_rounds=12):
     @staticmethod
     def check_constraints(constraints):
         for constraint in constraints:
-            if constraint.link is not None and not constraint.editable:
+            if constraint.link is not None and not constraint.editable and not (constraint.is_wheel or constraint.is_artifact):
```

Comment:
What does `is_artifact` add to this check? (Asking because the title says this resolves wheels.)

Reply:
Ah yes. I set this up because I wasn’t sure if we wanted to resolve artifacts too. Thoughts?

Reply:
changed and pushed, will let it build and then merge